### DISCUSSION TITLE
Parameter 'xxxx' input contains a handlebars binding - this is not allowed.

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#14662] [Help]
Hi Devs,

I am using following Binding to set value at a custom SQL query parameter: 
{{ date FilterOfferingsForm.Fields.inv_exempt_req_apr_date_picker "DD-MM-YYYY" }}

This was accepted at Budibase 2.19.5. After updating app to 2.32.8. I am getting the error: 
Parameter 'invexemptapprdateAfter' input contains a handlebars binding - this is not allowed. 

See also attached image. 
![image](https://github.com/user-attachments/assets/bb33927c-c999-4e45-971e-774b15c9b8f4)

If I remove date handlebar function and just keep reference to Calendar date picker form field error goes away. BTW I formatted handlebar this way to make sure query always gets the date in expected format.

Please explain why I am getting this error.

Also the Custom query page link at your documentation take me to 404 error (https://docs.budibase.com/docs/custom-queries). Could you please check it?

**Comment by mikesealey** [comment]
Hey @gmentzik 

As I understand it - if the value cannot be found for that particular handlebars-helper, it will instead substitute it with the handlebars-helper itself, which might explain why you're seeing this error.

Please try using the handlebars-helper in a paragraph component to see what it outputs. Your answer might lie within.

**Comment by gmentzik** [comment]
Hi @mikesealey , Indeed the error goes away if I use a paragraph component as value field or in case of date calendar if I set a value at form. So it happens if calendar has no value.

BTW: https://docs.budibase.com/docs/custom-queries, still returns 404 page not found.

**Reply by mikesealey** [reply]
> the error goes away if I use a paragraph component as value field

I might not have been clear - I want you to try to output the value into a paragraph component so that we can see what it is we're working with. You might also want to prefix it with "My value is: ", because otherwise an empty value might not be visible at all.

As for the faulty like to the documentation you've found - can you tell us where you got it from? I suspect you're trying to get to https://docs.budibase.com/docs/data-sources#custom-queries

**Reply by gmentzik** [reply]
> > the error goes away if I use a paragraph component as value field
> 
> I might not have been clear - I want you to try to output the value into a paragraph component so that we can see what it is we're working with. You might also want to prefix it with "My value is: ", because otherwise an empty value might not be visible at all.
> 
> As for the faulty like to the documentation you've found - can you tell us where you got it from? I suspect you're trying to get to https://docs.budibase.com/docs/data-sources#custom-queries

Hi @mikesealey ,
This is the result of the test your suggested:
![image](https://github.com/user-attachments/assets/a5a13840-51f9-4ea8-991a-a370f063c4df)

If the output of date calendar component is always ISO format  YYYY-MM-DD, then I may drop the conversion anyway. Can you confirm this?

Regarding the wrong link to custom sql paragraph  it can be found at this page: https://docs.budibase.com/docs/mysql-mariadb

> Whilst we could use multiple nested repeaters to pull in all of the department detail we want to display, it is much more efficient to use a [Custom SQL query](https://docs.budibase.com/docs/custom-queries).
